docs/concepts/providers.md

```diff
@@ -7,41 +7,34 @@ indent: true
 
 # Providers
 
-Providers extend Crossplane to enable infrastructure resource provisioning. In
-order to provision a resource, a Custom Resource Definition (CRD) needs to be
-registered in your Kubernetes cluster and its controller should be watching the
-Custom Resources those CRDs define. Provider packages contain many Custom
-Resource Definitions and their controllers.
+Providers are Crossplane packages that bundle a set of [Managed
+Resources][managed-resources] and their respective controllers to allow
+Crossplane to provision the respective infrastructure resource.
 
 Here is the list of prominent providers:
 
-### AWS Provider
+|   |   |   |   |
+|---|---|---|---|
+| Provider AWS  |  [GitHub Repo](https://github.com/crossplane/provider-aws) | [API Reference](https://doc.crds.dev/github.com/crossplane/provider-aws)  |  [Amazon Web Services (AWS) IAM User] |
+| Provider Jet AWS  |  [GitHub Repo](https://github.com/crossplane-contrib/provider-jet-aws) | [API Reference](https://doc.crds.dev/github.com/crossplane-contrib/provider-jet-aws) |   | 
+| Provider GCP |  [GitHub Repo](https://github.com/crossplane/provider-gcp) | [API Reference](https://doc.crds.dev/github.com/crossplane/provider-gcp) |   [Google Cloud Platform (GCP) Service Account] | 
+| Provider Jet GCP  |  [GitHub Repo](https://github.com/crossplane-contrib/provider-jet-gcp) | [API Reference](https://doc.crds.dev/github.com/crossplane-contrib/provider-jet-gcp)  |   | 
+| Provider Azure  | [GitHub Repo](https://github.com/crossplane/provider-azure) | [API Reference](https://doc.crds.dev/github.com/crossplane/provider-azure)  |  [Microsoft Azure Service Principal] | 
+| Provider Jet Azure  |  [GitHub Repo](https://github.com/crossplane-contrib/provider-jet-azure) | [API Reference](https://doc.crds.dev/github.com/crossplane-contrib/provider-jet-azure) |   | 
+| Provider Alibaba |  [GitHub Repo](https://github.com/crossplane/provider-alibaba) | [API Reference](https://doc.crds.dev/github.com/crossplane/provider-alibaba)  |   | 
+|   |   |   |   |
```

Comment:
Is this line an intentional separation of providers?

Reply:
Yes. It separates the 3(4) big cloud providers from the others.

Reply:
Makes sense 👍🏻 I wonder what our criteria for this distinction would be -- for instance, I could see if being providers under Crossplane org (rather than crossplane-contrib), but that would exclude the new jet providers. The reason why this comes to mind is that many of the ones below the line are more actively maintained than `provider-alibaba`. I don't think anything necessarily needs to change here, but could be some motivation for defining tiers of support more explicitly in the future :)